README.md

```diff
@@ -1,161 +1,245 @@
+
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
 
-Setup
 Clone this repository to your desired folder:
 
-Install
-Install this project with:
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
+
+### Install
+Install this project with these cmd:
+
+     1.cd Desktop/
+     2git clone using repository link
+     3.cd Hello-world/
+     4.git branch branch_name
+     5.git checkout branch_name
 
-Usage
-To run the project, execute the following command:
+gitbash and clone the repo using HTTP link
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+<!-- ### Usage
+
+To run the project, execute the following command: -->
+
+<!--
+Example command:
+
+```sh
+  rails server
+```
+--->
+
+### Run tests
 
-Run tests
 To run tests, run the following command:
 
-Deployment
-You can deploy this project using:
+     1.Run tests
+     2.To run tests run :
+     3.npx hint .
+     4.npx stylelint "*/.{css,scss}"
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
 
-(back to top)
+<!-- ### Deployment
 
-👥 Authors
-Mention all of the collaborators of this project.
+You can deploy this project using: -->
 
-👤 Author1
+<!--
+Example:
 
-GitHub: @techmoves
-Twitter: @Dopezonke_
-LinkedIn: https://www.linkedin.com/in/phelelani-ndlovu-883916160/
-👤 Author2
+```sh
 
+```
+ -->
 
-(back to top)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-🔭 Future Features
-Describe 1 - 3 features you will add to the project.
+<!-- AUTHORS -->
 
- [new_feature_1]
- [new_feature_2]
- [new_feature_3]
-(back to top)
+## 👥 Authors <a name="authors"></a>
 
-🤝 Contributing
-Contributions, issues, and feature requests are welcome!
+> Mention all of the collaborators of this project.
 
-Feel free to check the issues page.
+👤 **Phelelani**
 
-(back to top)
+- GitHub: [@githubhandle](https://github.com/techmoves)
+- Twitter: [@twitterhandle](https://twitter.com/Dopezonke_)
+- LinkedIn: [LinkedIn][(https://linkedin.com/in/phelelani-ndlovu-883916160)]
 
-⭐️ Show your support
-if you find it intersting let us know we are open for collarboration
+
+
+
+
+<!-- SUPPORT -->
+
+
+
+> Write a message to encourage readers to support your project
+> to new student    from microverse they veiw this project for review
 
 If you like this project...
 
-(back to top)
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank... microverse and the whole team
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
```

Comment:
- [ ] FAQ section is optional in the project README. Please 🙏 remove it from your README if you don't want put details 😃